src/Raven.Studio/typescript/models/wizard/unsecureSetup.ts

```diff
@@ -28,12 +28,28 @@ class unsecureSetup {
     }
     
     private initValidation() {
+        const currentHttpPort = window.location.port || "80";
```

Comment:
Technically, they can run the setup while using HTTPS if they re-run by editing the `settings.json` to say `SetupMode=Initial`. Can we add the check for 443 here?

Reply:
Yes

Reply:
```
const currentHttpPort = window.location.port || (window.location.protocol === "https:" ? "443" : "80")
```

Reply:
Yes